### ISSUE TITLE
Cannot change vertical_whitespace rule severity to error

### ISSUE THREAD

**Issue by maveric94** [realm/SwiftLint#2181]
### New Issue Checklist

- [✔️] Updated SwiftLint to the latest version
- [✔️] I searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues)

I want to change vertical_whitespace rule severity to error.

### Environment

* SwiftLint version 0.25.1
* Installation method CocoaPods

```yml
disabled_rules: # rule identifiers to exclude from running
  - force_cast
  - trailing_whitespace
  - unused_closure_parameter
excluded:
  - Operations/Operations

closure_parameter_position: error

vertical_whitespace:
  - severity: error
  - max_empty_lines: 2

void_return: error

trailing_newline: error
```

* Are you using [nested configurations](https://github.com/realm/SwiftLint#nested-configurations)?
No
* Xcode version 9.3

```swift
// This triggers a warning, not an error:
class A {
    
}



class B {
    
}
```

Also noticed same problem with statement_position rule.

**Comment by maveric94** [comment]
I figured out that I dont have to add dashes to rule config, ie
```yml
vertical_whitespace:
  max_empty_lines: 1
  severity: error  
```
